src/sidebar/services/groups.js

```diff
@@ -366,7 +366,10 @@ export default function groups(
       g => g !== null
     );
 
-    addGroupsToStore(groups);
+    // Optional direct linked group id. This is passed in the Notebook context.
```

Comment:
I'm left feeling unsure what "passed" means in this comment.

Reply:
perhaps "used" would be better